build.gradle

```diff
@@ -136,11 +136,31 @@ publishing {
                 groupId = "org.opensearch.plugin"
             }
         }
+        mavenJava(MavenPublication){ publication ->
```

Comment:
nit: with the other publication called `pluginZip`, do you think we should call this `pluginJar`?

Reply:
done